robot_skills/src/robot_skills/amigo.py

```diff
@@ -1,9 +1,33 @@
 #! /usr/bin/env python
 
 import robot
+import rospy
+import tf
+import math
+
+from geometry_msgs.msg import PoseStamped
+
+
+class SSL(object):
```

Comment:
I suggest to refactor it to a robotpart. Or at least move it to somewhere else. Because it will be used on multiple robots.

Reply:
@Rayman will handle this